src/pages/task/TaskTabs.tsx

```diff
@@ -115,7 +115,7 @@ export const TaskTabs: React.FC<TaskTabProps> = ({ isDisplayTask, task }) => {
         data-cy="task-files-tab"
         key="task-files-tab"
       >
-        <FilesTables />
+        <FilesTableTab taskId={id} execution={execution} />
       </Tab>
```

Comment:
I'm not 100% sold on the name `"FilesTableTab"` since the real `Tab` seems to be the component wrapping this 🤔